src/promnesia/sources/auto.py

```diff
@@ -185,6 +186,10 @@ def index(
                 logger.debug('detected %s as Obsidian vault', root)
                 replacer = obsidian_replacer
 
+        if root is not None and (root / "logseq").exists():
```

Comment:
a minor thing, but maybe worth detecting "logseq/config.edn"? Since currently this would match a `logseq` code repository, which is not we want here

Reply:
sounds good